src/components/Nav.jsx

```diff
@@ -12,9 +12,11 @@ import {
 } from '@material-ui/core';
 import { ChevronLeft, Menu } from '@material-ui/icons';
 import '../css/Nav.css';
+import { useSelector } from 'react-redux';
+import { selectLogIn } from '../features/userSlice.js';
 
 // eslint-disable-next-line react/prop-types
```

Comment:
do not use es-lint disable

Reply:
Changes have been made and will reflect in our next PR